index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href= "styles/index.css"/>
+</head>
+<body class="parent-side-container">
+    <header> 
+        <h1> Weather Report </h1>
+        <h2> 🌈 For the lovely city of <span class="city"> Gayville </span> 🌈 </h2>
+    </header>
+    
+    <section class = "temperature-side-container" action="#"> 
+        <h3> Temperature </h3>
+        <p id = "temperature-value"> 76 </p>  
```

Comment:
Minor note: it is a convention not to include spaces between a properties and it's value.

```suggestion
        <p id="temperature-value"> 76 </p>  
```